db/migrate/20190423235125_create_join_table_users_recipes.rb

```diff
@@ -0,0 +1,8 @@
+class CreateJoinTableUsersRecipes < ActiveRecord::Migration
+  def change
+    create_join_table :users, :recipes do |t|
```

Comment:
Does this just create an intermediate table that joins users and recipes?

Reply:
Yes!